test/app/features/help_center/data/datasources/guardian_datasources_delete_test.dart

```diff
@@ -80,29 +89,29 @@ void main() {
 
       group('delete()', () {
         test(
-          'should perform a DELETE with X-API-Key',
+          'perform a DELETE with X-API-Key',
           () async {
             // arrange
-            final endPointPath = '/me/guardioes/${guardian!.id}';
+            final endPointPath = '/me/guardioes/${guardian.id}';
             final headers = await _setUpHttpHeader();
-            final request = _setuHttpRequest(endPointPath, {});
+            final request = _setUpHttpRequest(endPointPath, {});
             _setUpMockPostHttpClientSuccess204();
             // act
             await dataSource.delete(guardian);
             // assert
-            verify(apiClient.delete(request, headers: headers));
+            verify(() => apiClient.delete(request, headers: headers)).called(1);
           },
         );
         test(
-          'should get a valid ValidField for a successful delete',
+          'get a valid ValidField for a successful delete',
           () async {
             // arrange
             _setUpMockPostHttpClientSuccess204();
-            const expected = ValidField();
+            const actual = ValidField();
             // act
             final received = await dataSource.delete(guardian);
             // assert
-            expect(expected, received);
+            expect(actual, received);
```

Comment:
Os argumentos do `expect` estão invertidos, da maneira que esta caso o teste falhe a mensagem de erro será confusa, mostrando que o valor retornado como se fosse o correto (esperado).
```suggestion
            const expected = ValidField();
            // act
            final received = await dataSource.delete(guardian);
            // assert
            expect(received, expected);
```